muxml.cpp

```diff
@@ -139,14 +139,16 @@ bool MuXML::read(std::istream& in,XMLNode* parent,const int& level,const char& c
    
    int index = 0;
    bool success = true;
+   bool xml_found = false;
    char c = currentChar;
    char buffer[1024];
    do {
       // Skip empty chars:
       while ((c == ' ' || c == '\t' || c == '\n') && in.good() == true) in >> c;
-      
+	  if(in.good() == false){ return false;}
```

Comment:
Indentation!

Reply:
Fixed, but the whole file could do with a whitespace makeover.